### PR TITLE
inital commit

### PR DESCRIPTION
**Pull request by sabioyo**

### CODE REVIEW

File: sabi/styles.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
+
+.content {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
```
Comment (Jeremiahiro) [inline sabi/styles.css:4]: You need two columns, why are you splitting template-columns into four?

File: sabi/styles.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    margin: 20px auto;
+  }
+  .logo {
+    display: grid;
+    align-items: flex-start;
+    width: fit-content;
+  }
+.content div {
+    border: 2rem;
+    display:flexbox;
```
Comment (Jeremiahiro) [inline sabi/styles.css:32]: flexbox is not a flex property

File: sabi/styles.css [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  }
+  .two {
+    width: 425px;
+    height: 100%;
+    background-color: #efb762;
+    margin: 20px auto;
+  }
+  .logo {
+    display: grid;
+    align-items: flex-start;
```
Comment (Jeremiahiro) [inline sabi/styles.css:27]: flex and grid do not go together

**Review by Jeremiahiro — CHANGES_REQUESTED** [review]
Screenshot 2023-02-04 at 21 33 23

- No social icon
- too much use of margin
-  wrong combination and use of flex and grid property

**Comment by Jeremiahiro** [comment]
Well done but you are not there yet. You missed a few closing div tags and a few divs that are not necessary. The issues raised in the previous review have also not been addressed.

**Comment by sabioyo** [comment]
changes have been made.

**Pull request merged**